tools/update_stable_tags.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import subprocess
+import yaml
+import logging
+from datetime import datetime
+
+logging.basicConfig(format='%(asctime)s %(message)s')
+logging.getLogger().setLevel(logging.INFO)
+
+#######################################################################
+# This will update the stable (by default) tag to the current revision
+# +  A tag <tag>.%d/%m/%Y-%H-%M will mark the commit
+# notice: * repo has valid push credentials
+#         * working location must be  from in the cloned item
+#######################################################################
+
+retagging_commands = [
+            "git push --delete origin {tag} || true",
+            "git tag -d {tag} || true",
+            "git tag -f {tag}",
+            "git tag -f {tag}.{date}",
+            "git push --tags origin {tag}",
+            ]
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--tag", help="assisted_installer_tag", type=str,
```

Comment:
Since in the main usage of the script we'll need to push multiple tags (stable, stable.<data>) maybe it should accept multiple tags arguments and do them at once?